### ISSUE TITLE
WCAG AA conformance

### ISSUE THREAD

**Issue by matthewdarwin** [canada-ca/PCO-Public-Opinion-Research__Recherche-en-opinion-publique--BCP#4]
This proposal requests documents created are to be aligned to WCAG AA conformance. Reading the 
examples, it seems that not all the details needed to conform to WCAG AA are contained within the examples themselves.  Where should the additional information be obtained from?   Do we just invent it for the purposes of doing an example submission?

It seems to me that the requirements for (1) making a process repeatable in the future (presumably with the goal of as little manual intervention as possible), and (2) making the specific examples 100% WCAG AA compliant are not achievable.   

Building WCAG AA compliant documents requires an full understanding of the document by a human and if that is done, then how will that apply to a future document, which probably is not exactly the same format?

**Comment by mgifford** [comment]
As far I understand this what's being asked for is to have someone code something for $10k that will automatically produce accessible copies of reports and summary tables.

Given a brief look at the documents in question I'm not sure that this could be achieved for $10M. I'd certainly give it a try, but heck, just setting up contracts with government usually takes about $5k in effort. 

IT is powerful, but it isn't magic.

**Comment by steve-h** [comment]
Many of the tables in the doc files have complex dimensions and bridged cell headers.  These are hard to make accessible and likely not readable by screen readers as a result. WCAG AA would be hard to craft by hand without building separate tables that could be read.
Is it required to be an html file that accurately mimics the original report(easily done with existing online tools) or is future accessibility mandatory?  Best answer, as mentioned elsewhere, would be to get EKOS to bid to fix their original reports by submitting in the required format.

**Comment by steve-h** [comment]
This issue is still totally Open.  I decided to bid (and win) a $10K best effort solution. If I am not paid for missing the specification despite releasing my best efforts in open source, then it will be a learning in the "Development Exchange" part of the process.  @mgifford  is correct , I invested $5K in my bid, that was a risk investment. I don't think these small jobs deserve the waste of multiple talented bidders using their efforts to do the same investigations. Just open the item  for discussion, like on this thread, and then call for a show of hands and select one randomly. If they fail or disappoint in delivering a solution then select another developer to redo or finish the work.

I leave paying for the failure to the proposer.  They may wish to use a future weighting factor on the failed bidder based on their satisfaction.  I am also exposed to this risk by winning this technically impossible problem.

There is also no procurement vehicle to pay for ongoing maintenance, by anyone, for this repo. How do you add a couple of WCAG 2.0 AA tweaks that I may have ignored, yet are technically possible?

Oops, I opened more issues while trying to close this one.

**Comment by mgifford** [comment]
Maintenance is so hard. Nobody wants to pay for it. Just isn't sexy. That said, without maintenance, there is chaos. 

I would hope with a Modular approach https://modularcontracting.18f.gov/ there would be more room for this.  Issues could be added to the issue queue.  A public servant could review & rank the concerns, those could be grouped together into a package of issues and then someone could bid on them. 

If they are for $5-10k contacts, maybe more than one person or firm could win.  If there are multiple people bidding on the same work, a sufficiently technical government employee could evaluate them.  Ideally the work could be ranked for security, accessibility, documentation, performance (and other stuff that nobody wants to pay for). 

The government needs a reputation system. There is talk about dropping Buy & Sell for an "Amazon styled" procurement portal. The key element to Amazon (as I see it) is the reputation system. I've heard no discussion of this though.
